microsoft-iis-mixin/signals/overview.libsonnet

```diff
@@ -0,0 +1,216 @@
+        unit: 'Bps',
+        sources: {
+          prometheus: {
+            expr: 'windows_iis_sent_bytes_total{%(queriesSelector)s, site=~"$site"}',
+            legendCustomTemplate: '{{instance}} - {{site}}',
+          },
+        },
+      },
+
+      bytesReceived: {
+        name: 'Bytes received per second',
+        type: 'counter',
+        description: 'The traffic received by an IIS site.',
+        unit: 'Bps',
+        sources: {
+          prometheus: {
+            expr: 'windows_iis_received_bytes_total{%(queriesSelector)s, site=~"$site"}',
+            legendCustomTemplate: '{{instance}} - {{site}}',
+          },
+        },
+      },
+
+      filesSent: {
+        name: 'Files sent per second',
+        type: 'counter',
+        description: 'The files sent by an IIS site.',
+        unit: 'files',
+        sources: {
+          prometheus: {
+            expr: 'windows_iis_files_sent_total{%(queriesSelector)s, site=~"$site"}',
+            rangeFunction: 'increase',
+            legendCustomTemplate: '{{instance}} - {{site}}',
+          },
+        },
+      },
+
+      filesReceived: {
+        name: 'Files received per second',
+        type: 'counter',
+        description: 'The files received by an IIS site.',
+        unit: 'files',
+        sources: {
+          prometheus: {
+            expr: 'windows_iis_files_received_total{%(queriesSelector)s, site=~"$site"}',
+            rangeFunction: 'increase',
+            legendCustomTemplate: '{{instance}} - {{site}}',
+          },
+        },
+      },
+
+      // Connection signals
+      currentConnections: {
+        name: 'Current connections',
+        type: 'gauge',
+        description: 'The number of current connections to an IIS site.',
+        unit: 'none',
+        sources: {
+          prometheus: {
+            expr: 'windows_iis_current_connections{%(queriesSelector)s, site=~"$site"}',
+            legendCustomTemplate: '{{instance}} - {{site}}',
+          },
+        },
+      },
+
+      connectionAttempts: {
+        name: 'Connection attempts',
+        type: 'counter',
+        rangeFunction: 'increase',
+        description: 'The number of attempted connections to an IIS site.',
+        unit: 'none',
+        sources: {
+          prometheus: {
+            expr: 'windows_iis_connection_attempts_all_instances_total{%(queriesSelector)s, site=~"$site"}',
+            legendCustomTemplate: '{{instance}} - {{site}}',
+          },
+        },
+      },
+
+      // Server cache signals
+      fileCacheHitRatio: {
+        name: 'File cache hit ratio',
+        type: 'raw',
+        description: 'The current file cache hit ratio for an IIS server.',
+        unit: 'percent',
+        sources: {
+          prometheus: {
+            expr: 'increase(windows_iis_server_file_cache_hits_total{%(queriesSelector)s}[$__interval:]) / clamp_min(increase(windows_iis_server_file_cache_queries_total{%(queriesSelector)s}[$__interval:]),1) * 100',
```

Comment:
Same comment about `[$__interval:]`, it should be `[$__interval:] offset $__interval`